engine/esma_template.c

```diff
@@ -985,6 +939,8 @@ static int _decode_dbuf(char *line, struct esma_template *et)
 						__func__, et->name, p - 3);
 				goto __fail;
 			}
+			
+			state = st_trans_semicolon;
```

Comment:
What about `SIGQQQ`? 
With any signal name, the parser will **always** continue to work.